opencensus-migration/src/main/java/SpanConverter.java

```diff
@@ -0,0 +1,128 @@
+import io.opencensus.common.Function;
+import io.opencensus.implcore.trace.RecordEventsSpanImpl;
+import io.opencensus.trace.Annotation;
+import io.opencensus.trace.AttributeValue;
+import io.opencensus.trace.EndSpanOptions;
+import io.opencensus.trace.Link;
+import io.opencensus.trace.Span;
+import io.opencensus.trace.SpanContext;
+import io.opencensus.trace.TraceOptions;
+import io.opencensus.trace.Tracestate;
+import io.opencensus.trace.export.SpanData;
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.trace.Span.Builder;
+import io.opentelemetry.trace.SpanId;
+import io.opentelemetry.trace.TraceFlags;
+import io.opentelemetry.trace.TraceId;
+import io.opentelemetry.trace.TraceState;
+import io.opentelemetry.trace.Tracer;
+import java.util.EnumSet;
+import java.util.Map;
+import java.util.Map.Entry;
+
+public class SpanConverter {
+
+  public static class FakeSpan extends Span {
```

Comment:
I've left the FakeSpan class in here for now since it's only used in SpanConverter, but happy to move it out to its own file